core/services/pipeline/getters.go

```diff
@@ -147,7 +147,7 @@ func JSONWithVarExprs(jsExpr string, vars Vars, allowErrors bool) GetterFunc {
 				}
 				keypath, is := maybeKeypath.(string)
 				if !is {
-					return nil, errors.Wrapf(ErrBadInput, fmt.Sprintf("you cannot use %s in your JSON", chainlinkKeyPath))
+					return nil, errors.Wrap(ErrBadInput, fmt.Sprintf("you cannot use %s in your JSON", chainlinkKeyPath))
```

Comment:
Wouldn't we want to use `fmt.Errorf()` here?
```suggestion
					return nil, fmt.Errorf("you cannot use %s in your JSON: %w", chainlinkKeyPath, ErrBadInput)
```

Reply:
:shrug: I would love to remove `pkg/errors` everywhere, but there are objections from folks who want stack traces on all errors.

Reply:
No worries in that case